contributors/working_with_datastructures.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: documentation
+title:  Working with Datastructures
+order: 2
+---
+
+## 
+
+Check out the example on how to work with a table [here](https://github.com/Caleydo/lineage/blob/master/src/dataExplorations.ts). 
```

Comment:
@alexsb, we can point to https://github.com/Caleydo/phovea_demos/blob/master/src/UsingTable.ts now, right?